neqo-crypto/src/replay.rs

```diff
@@ -15,12 +15,7 @@ use std::ptr::null_mut;
 use std::time::{Duration, Instant};
 
 // This is an opaque struct in NSS.
-#[allow(
-    unknown_lints,
-    renamed_and_removed_lints,
-    clippy::unknown_clippy_lints,
-    clippy::upper_case_acronyms
-)] // Until we require rust 1.51.
+#[allow(clippy::upper_case_acronyms)] // Until we require rust 1.51.
```

Comment:
```suggestion
#[allow(clippy::upper_case_acronyms)]
```

We already require 1.51 :)